cibuild.sh

```diff
@@ -32,12 +44,16 @@ THIS_SCRIPT_PATH="`dirname \"$0\"`"
 PACKAGES_DIR="$THIS_SCRIPT_PATH/packages"
 MSBUILD_EXE="$PACKAGES_DIR/mono-msbuild/bin/Unix/Debug-MONO/MSBuild.exe"
 MSBUILD_DOWNLOAD_URL="https://github.com/Microsoft/msbuild/releases/download/mono-hosted-msbuild-v0.1/mono-msbuild.zip"
+MSBUILD_ZIP="$PACKAGES_DIR/msbuild.zip"
+HOME_DEFAULT="/tmp/msbuild-CI-home"
```

Comment:
@mmitche We're setting HOME if it's not set. Does this location look alright to you? Is it persisted from one run to the other? If it's not, it would increase the job time, since nuget will restore without a cache each time.